### PR TITLE
Update wg.md

### PR DESCRIPTION
**Pull request by marcoradici**
Updated structure of working groups, charges, membership.

### CODE REVIEW

File: _content/wg.md [hunk 1]
```diff
@@ -42,46 +44,67 @@ the 2nd detector should be complementary to the project detector at IR6 and may
...
+---
+
+
+### Artificial Intelligence
+
+The mission of the Artificial Intelligence (AI) Working Group is to develop and maintain connections to the data science community, and engaging with the rapidly evolving AI/ML  (Machine Learning) toolset that may impact the realization of the EIC science mission. The AI will organize events to educate and assist the EIC community in utilizing AI/ML techniques in the area of detector design and controls, simulations, data readout and analysis, and theory and phenomenology. The AI will also serve as an [entry point to AI applications](https://eic.ai){:target="_blank"}. 
+
+The conveners are:
+*  Tanja Horn (CUA, USA)  <hornt@cua.edu>
+*  Cristiano Fanelli (William and Mary University, USA) <cfanelli@wm.edu>
```
Comment (wdconinc) [inline _content/wg.md:95]:
```suggestion
*  Cristiano Fanelli (William & Mary, USA) <cfanelli@wm.edu>
```

File: _content/wg.md [hunk 2]
```diff
@@ -96,6 +119,27 @@ The EICUG community currently supports and is using the following collaborative
...
 
-If you have questions or need assistance with the EIC Collaborative Tools please contact
-the coordinator: potekhin_at_bnl.gov.
+If you have questions or need assistance with the EIC Collaborative Tools, please contact the coordinator: <potekhin@bnl.gov> .
+
+<!-- [Contacting the conveners](eic-det2-conveners-l@lists.bnl.gov){:target="_blank"} 
+
+<span id="software"><br/></span>
+
+---
```
Comment (wdconinc) [inline _content/wg.md:128]: Illegal double dash in comment. Ref. https://www.w3.org/TR/REC-xml/#sec-comments

Also, comment not closed.

Comment (buddhasystem) [reply _content/wg.md]: What are we going to do about this item?

Comment (buddhasystem) [reply _content/wg.md]: That span item... It's there for navigation? Was it removed? I think I put it there for anchor purposes a while ago.

Comment (marcoradici) [reply _content/wg.md]: It belongs to the previous piece of text describing the old structure, that I commented and kept as history log. To my knowledge, it should break any anchor link from outside/inside this document.

Comment (wdconinc) [reply _content/wg.md]: History shouldn't be kept as comments in the files. We use git because it keeps history for us.

**Review by wdconinc — COMMENTED** [review]
So the EICUG has completely removed the Software WG? Or is that an editing mistake?

**Comment by buddhasystem** [comment]
> So the EICUG has completely removed the Software WG? Or is that an editing mistake?

AFAIK indeed the Software WG is now officially decommissioned. It's now in ePIC.

**Review by wdconinc — APPROVED** [review]

**Comment by wdconinc** [comment]
> AFAIK indeed the Software WG is now officially decommissioned. It's now in ePIC.

It does seem a little strange to remove this entirely while other past working group leadership goes into _content/previous_leadership.md.

**Comment by buddhasystem** [comment]
Wouter -- I agree. Up to the leadership to decide. It's a special case I guess since the whole group migrated to ePIC.

**Comment by marcoradici** [comment]
In _content/previous_leadership.md you only find the history of leadership, because the corresponding working groups still exist. Apart of archiving purposes, it's a memory to help rotating names in a balanced way. The EICUG Software WG instead does not exist anymore. According to Phase 3, with the creation of ePIC the group moved to ePIC. As you say, github keeps memory of the past structure for us...

**Comment by wdconinc** [comment]
There was no "migration of this group to ePIC": three of the SWG conveners resigned their long-term EICUG roles because of increasing demands both in ePIC and outside of ePIC. There were still two other SWG conveners who continue in what is now the EICUG AI working group. People resign convener roles all the time; that doesn't mean that their WG migrates with them. There are many in the EICUG who are interested in software who did not migrate to ePIC. Nothing in the phase 3 charter specifies that the SWG "migrates to ePIC": it is an adhoc WG in EICUG that was not continued. My concern is that other former adhoc WGs are reflected in the history of leadership. It seems like an intentional snub not to do that with the SWG conveners.

**Comment by buddhasystem** [comment]
Wouter -- thanks for clarifying all these points -- these are excellent arguments. I think it's in the hands of the Steering Committee.

**Comment by buddhasystem** [comment]
Dear All, sorry for adding my own confusion to this thread. I'm sure this will be sorted out, I just misinterpreted some of the info.

**Pull request merged**